### PR TITLE
[PR] Keyword Argument Fix, Transform Win/Loss dictionary fix

### PR DESCRIPTION
**Pull request by scmisa**
- economy average vs outcome result dicts
- mmr_vs_result transform incomplete dict 
- utils.select_outcome_1v1 result dict incomplete

average_players_stats function parameter changed and in old code keyword wasn't changed
from `average_player_features = average_player_stats(
        player_tracker_events=sc2_replay.trackerEvents
    )`
to `average_player_features = average_player_stats(
        sc2_replay=sc2_replay
    )`

### CODE REVIEW

**Comment by scmisa** [comment]
**_

> I have read the CLA Document and I hereby sign the CLA

_**

**Comment by scmisa** [comment]
> _**I have read the CLA Document and I hereby sign the CLA**_

**Comment by scmisa** [comment]
_**I have read the CLA Document and I hereby sign the CLA**_

**Comment by scmisa** [comment]
recheck

**Comment by scmisa** [comment]
**recheck**

File: .python-version [hunk 1]
```diff
@@ -1 +1 @@
-3.11.4
+3.11.3
```
Comment (Kaszanas) [inline .python-version:1]: Can you elaborate on this change? Why did the Python version change?

**Review by Kaszanas — COMMENTED** [review]
There is one comment that requires clarification.
As far as the rest of the code goes if it passes the CI, it should be good to go.

**Comment by Kaszanas** [comment]
recheck

**Pull request closed without merging**